api/api/spec/spec.yaml

```diff
@@ -4133,6 +4133,8 @@ components:
             $ref: '#/components/schemas/LogSummary'
           wazuh-modulesd:syscollector:
             $ref: '#/components/schemas/LogSummary'
+          wazuh-modulesd:vulnerability-scanner:
+            $ref: '#/components/schemas/LogSummary'
```

Comment:
Indeed, there are some new additional categories inside `wazuh-modulesd` (for example `wazuh-modulesd:content_manager`) and outside (`indexer-connector`) which we could add too. This is a snippet of my 4.8.0 local env:
```
2024/02/09 09:53:37 wazuh-modulesd:content_manager: INFO: Starting content_manager module.
2024/02/09 09:53:37 wazuh-modulesd:database: INFO: Module started.
2024/02/09 09:53:37 wazuh-modulesd:download: INFO: Module started.
2024/02/09 09:53:37 wazuh-modulesd:control: INFO: Starting control thread.
2024/02/09 09:53:37 sca: INFO: Starting evaluation of policy: '/var/ossec/ruleset/sca/cis_ubuntu22-04.yml'
2024/02/09 09:53:37 wazuh-modulesd:syscollector: INFO: Module started.
2024/02/09 09:53:37 wazuh-modulesd:syscollector: INFO: Starting evaluation.
2024/02/09 09:53:37 indexer-connector: WARNING: No username and password found in the keystore, using default values.
2024/02/09 09:53:37 wazuh-modulesd:syscollector: INFO: Evaluation finished.
2024/02/09 09:53:37 indexer-connector: WARNING: Error initializing IndexerConnector: Couldn't connect to server, we will try again after 2 seconds.
2024/02/09 09:53:37 wazuh-modulesd:vulnerability-scanner: ERROR: VulnerabilityScannerFacade::start: Couldn't find column family: 'vendor_map'
2024/02/09 09:53:37 wazuh-syscheckd: INFO: (6009): File integrity monitoring scan ended.
2024/02/09 09:53:37 wazuh-syscheckd: INFO: FIM sync module started.
2024/02/09 09:53:39 indexer-connector: WARNING: Error initializing IndexerConnector: Couldn't connect to server, we will try again after 4 seconds.
2024/02/09 09:53:42 sca: INFO: Evaluation finished for policy '/var/ossec/ruleset/sca/cis_ubuntu22-04.yml'
2024/02/09 09:53:42 sca: INFO: Security Configuration Assessment scan finished. Duration: 5 seconds.
2024/02/09 09:53:43 indexer-connector: WARNING: Error initializing IndexerConnector: Couldn't connect to server, we will try again after 8 seconds.
2024/02/09 09:53:51 indexer-connector: WARNING: Error initializing IndexerConnector: Couldn't connect to server, we will try again after 16 seconds.
2024/02/09 09:53:54 rootcheck: INFO: Ending rootcheck scan.
2024/02/09 09:54:07 indexer-connector: WARNING: Error initializing IndexerConnector: Couldn't connect to server, we will try again after 32 seconds.
```